pkg/manager/member/orphan_pods_cleaner.go

```diff
@@ -89,6 +118,9 @@ func (opc *orphanPodsCleaner) Clean(tc *v1alpha1.TidbCluster) (map[string]string
 			return skipReason, err
 		}
 
+		// if the PVC is not found in apiserver (also informer cache) and the
+		// phrase of the Pod is Pending, delete it and let the stateful
+		// controller to create the pod and its PVC(s) again
 		err = opc.podControl.DeletePod(tc, pod)
```

Comment:
only delete pod if the pod status is `Pending`

Reply:
sorry, I forgot it...added in https://github.com/pingcap/tidb-operator/pull/878/commits/979117825df3ddd85232fd8a3f1354a67b80cfd3